scripts/autolink-ios.rb

```diff
@@ -56,5 +55,24 @@ def use_react_native! (options={})
     pod 'React-RCTFabric', :path => "#{prefix}/React"
     pod 'Folly/Fabric', :podspec => "#{prefix}/third-party-podspecs/Folly.podspec"
   end
+end
 
+def add_flipper_pods!
```

Comment:
@alloy Should we allow passing the flipper version here to override the default one, this would allow updating flipper independently of RN.

Reply:
Ah yes, good call; I had wanted to do that but clearly forgot, thanks!

Reply:
(Won’t make it for RC 3, but the next release.)